Makefile

```diff
@@ -563,34 +563,6 @@ endif
 
 uninstall: rbthemis_uninstall
 
-## Node.js #####################
```

Comment:
now we can't install jsthemis from root folder?

Reply:
Well, to be honest, `make jsthemis_install` is not of much use for the users. Yes, it installs JsThemis into `node_modules`... in Themis source tree root. You can't use it to build and install JsThemis from source into your own application's `node_modules` right now. And I don't really see _how_ it could be used at all.

With these changes `make jsthemis_install` works are before, installing JsThemis into node_modules in the current directory (which must be Themis source root).

I also don't really like the fact that we have to pollute the source tree with node_modules (the source may be mounted as read-only, with only build directory being writable), but I guess there is no way around that in JavsScript land.

I figured that maybe limiting it to the wrapper source directory will be better (as it allows to special mocha's version in one place), but that brings other set of issues: the tests currently `require('jsthemis')` which requires JsThemis to be installed into node_modules somewhere. The proper way to do this seems to be `npm link`, but for some reason it cannot be used twice because it removes other dependencies like `nan` from `node_modules`. I've been experimenting with this for some time since I've noticed that red build, but have not found a good solution to that for now.

Reply:
Actually I don't know ways how to install js package from sources to some app folder. Only by manual copying required folder or installing by tarball from app's folder. In common way all devsinstall dependencies from registry and package.json. So if someone wants to install it from sources, it is his responsibility to do it correct. Simplest way is to install it globally to system or at some place of a hierarchy of dev environment (https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders). So user should change folder to any folder in such hierarchy and copy here built jsthemis to use it in own app.

One moment where we can simplify installing is to use user's `npm` flags in our makefile and allow to pass `--global`/`--local` params to `npm install` to allow them install it globally, IMHO

Reply:
`--global` flag is mostly useful to [install binaries][1], not libraries. Most projects should use local installation to fetch only their dependencies and have something packageable in their local `node_modules`, not install a library globally and expect it to be available when the application is deployed. That's what `npm install` will do by default when a project has JsThemis in `package.json` as a dependency.

[1]: https://nodejs.org/en/blog/npm/npm-1-0-global-vs-local-installation/

Now I've add a `jsthemis` target that can be used to prepare a tarball with JsThemis from the source tree which can then be used by developers to install JsThemis manually in whatever way they want.